updating/updating_a_cluster/migrating-to-multi-payload.adoc

```diff
@@ -36,3 +36,4 @@ include::modules/migrating-to-multi-arch-cli.adoc[leveloffset=+1]
 * xref:../../updating/understanding_updates/understanding-update-channels-release.adoc#understanding-update-channels-releases[Understanding update channels and releases]
 * xref:../../installing/overview/installing-preparing.adoc#installing-preparing-selecting-cluster-type[Selecting a cluster installation type]
 * xref:../../machine_management/deploying-machine-health-checks.adoc#machine-health-checks-about_deploying-machine-health-checks[About machine health checks]
+* xref:../../updating/updating_a_cluster/updating-cluster-cli.adoc#update-upgrading-oc-adm-upgrade-status[Gathering cluster update status using oc adm upgrade status (Technology Preview)]
```

Comment:
Missing the assembly context in the anchor here (saw notifications in my email and decided to take a peek 😅)
```suggestion
* xref:../../updating/updating_a_cluster/updating-cluster-cli.adoc#update-upgrading-oc-adm-upgrade-status_updating-cluster-cli[Gathering cluster update status using oc adm upgrade status (Technology Preview)]
```